src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,42 @@
+.nav {
+  &__list {
+    margin-right: 50px;
+  }
+
+  &__item {
+    display: inline-block;
+    list-style: none;
+    padding: 11px 0;
+  }
+
+  &__link {
+    position: relative;
+    padding: 23px 0;
+    color: #000;
+    text-decoration: none;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 17px;
+  }
+
+  &__link:hover {
+    color: #00acdc;
+  }
```

Comment:
you can use scss nesting here 
`&__link {
      ....
      &:hover {
         ....
      }
  }
`